tiledb/libtiledb.pyx

```diff
@@ -3649,6 +3652,10 @@ cdef class Array(object):
         cap = PyCapsule_New(<void *>(self.ptr), name, NULL)
         return cap
 
+    def __repr__(self):
+        return "Array(uri={0}, mode={1}, ndim={2})"\
```

Comment:
Please add single-quote around the uri.

Reply:
OK. Using `uri={0!r}` which will include single-quotes and is consistent with other parts of the code.